DNDMiniInjector.lua

```diff
@@ -1922,6 +1930,8 @@ versionNumber = "4.7.9"
 finishedLoading = false
 debuggingEnabled = false
 pingInitMinis = true
+displayNPCInitiative = true
+displayInitiative = true
```

Comment:
I can understand how this naming may be confusing given I also introduced a variable named hideInitiative (intended for units specifically)